tools/esp32/Config.mk

```diff
@@ -22,9 +22,9 @@
 # and assemble source files and to insert the resulting object files into an
 # archive.  These replace the default definitions at tools/Config.mk
 
-ifdef BLOBDIR
-	BOOTLOADER=${BLOBDIR}/esp32core/bootloader.bin
-	PARTITION_TABLE=${BLOBDIR}/esp32core/partition-table.bin
+ifdef ESPTOOL_BINDIR
+	BOOTLOADER=${ESPTOOL_BINDIR}/bootloader.bin
+	PARTITION_TABLE=${ESPTOOL_BINDIR}/partition-table.bin
```

Comment:
```suggestion
	BOOTLOADER=$(ESPTOOL_BINDIR)/bootloader.bin
	PARTITION_TABLE=$(ESPTOOL_BINDIR)/partition-table.bin
```

Reply:
Will apply next time I push.

BTW, @gustavonihei I was thinking we should remove the part that gets the binaries from a hello-world example.  First it looks weird :) second, usually we flash the bootloader and partition-table only once and then we only need to re-flash the NuttX image.

So, will end up with `make download ESPTOOL_PORT` flashes only nuttx.  `make download ESPTOOL_PORT=.. BLOBDIR=.` flashes all three.

Reply:
Good observation! In fact the bootloader and partition only need to be flashed once. Our documentation and Sara tutorials needs to me adapted to warning the user. He/she only need to do it one time!

Reply:
We have to update those tutorials as well to include these new modifications

I'll update this PR in a little while.

Reply:
I am just waiting it to be merged to update my tutorials